### PR TITLE
fix: correct object being used to log

### PR DESCRIPTION
**Pull request by mohamed-cds**
This PR fixes logging so the correct object is used. Also introduces a new bash script to test the file upload process via local lambda running in the devcontainer.

We also need to disable the ClamAV cache since it'll result in a scan time of `0.000` making it indistinguishable from scenarios where files were not scanned due to file size, etc.

### CODE REVIEW

File: bin/file_upload_payload [hunk 1]
```diff
@@ -0,0 +1 @@
...1zU5Xh/4doC0KH16QYlU+n8B8JayKF4RBbWhcjCApzuufl8iAAZvkwjidZfThGs3Kz8DsP5rt/SThclsgm7SBej1JjR9xaJalIWS2xa0v0Kw6ysXMyH5Tlc4jr/+If5nAAZ7hV3G7UhkbcejIPsEROIiF2Se11uczOotSE+XJcLVy0hvS8tsGhoi1bGUNicaN9kj+ti7IZuVsLbtTekkCFF1Jct7igKNDSuz5NyaZPbS01sfALbbNHRCzwmi/oUN888AXHYFrud6cuGeScElZyfRIUmld8W6X5VoEn3kuQL7LwAI/6bRgAFcrk0ZzZmbbNNNdEsauWfHSz5SiMjBSZO8vENbFnygGDaNoMe/CIAs/2UBk6IUzjUtS23+jRL/GoB5r6R5ahqaniVGhxFkye1qZMQ6GXmIDuB4qE1ID/tqfbKrJ1F+Q9Asy1L8nebQTQBIEJKzsZLjAZC1yZFt0+k2I/yQM/EjsytYOYtsxFytD4XmUB0/pbc5zfDXr98o/RzAEE1ezdsdQUMzTZRhVlZV1S9faR/TkjRKRyVExWYJqn21PPkw6n4KdYJX/mYU+bpCBA3o6UkPgCXajlP1y9Lglewi+alDtMqaSJPJ418AZHSjZh2joWGoRufshtv1W57XvdQHAIyPRDtMIzbI63sAAuqN3f1O7pbITwA2R86wwA2IhJs+tBqySeB7jiUWcF1ZIb96SLKz5wUP40oC2N222C1Ustn29mqNuvrixo/vJx2yvWN75QddmPZ7Bb10QFui0nZIk+I0ETO4YJJ5rJcCkrcZYyNbFYQfOdBz80MsJctFiXdYlzWhyeWrfcb00MDR0cDAOr7OfxwcfDHxttXdOpC7iYs5wCEFS1AoStTdfs4RDXHQmdDVuHv57TUvVBowul6gMf5JrleqHFYXhFa/BxpLW3xX/11+4T7ShxyZjhGvMJarEVuY5GSh9TXvnrh+t/xo/sXg17yZdajjdWLUCkn1NJvEcyx/K0e7v/JKH4m0RLzv7WmSKipvfd07zm7trd/po04vFLwkPRi4tbd4fD3C0LxCdBDtCFsDt/tGp69nKCFLB0rf8p7L//q77/9XgAEAkNtlT7ZsGYcAAAAASUVORK5CYIINCi0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS03MzM5MjI5MTA1NTMzNTg5MzE3NjEyODAtLQ0K
```
Comment (patheard) [inline bin/file_upload_payload:1]: I'm deeply curious about what this is.

File: api/main.py [hunk 2]
```diff
@@ -44,7 +44,9 @@ def handler(event, context):
         if scanning_request_id:
             context.logger = CustomLogger(context.aws_request_id, scanning_request_id)
         else:
-            context.logger = log
+            context.logger = CustomLogger(
```
[the remark this answers is not in the document]
Comment (mohamed-cds) [reply api/main.py]: that's so much cleaner. Changed to what you suggested!

File: bin/file_upload_payload (hunk 1 above)
Comment (mohamed-cds) [reply bin/file_upload_payload:1]: base64 encoded file upload. Took me too long to figure out how files are being sent via lambda event payload.
image

Comment (patheard) [reply bin/file_upload_payload:1]: ah that explains why it went all funky on me when I tried `base64 -d`

**Review by patheard — APPROVED** [review]
⚡

**Pull request merged**